feature_tests/src/callbacks.rs

```diff
@@ -0,0 +1,46 @@
+#[diplomat::bridge]
+mod ffi {
+
+    pub struct Wrapper {
```

Comment:
nit: over time it becomes annoying to find the right opaques for the right tests, let's call this CallbackWrapper and CallbackTestingStruct